tests/rptest/tests/rpk_cluster_test.py

```diff
@@ -221,8 +221,7 @@ def get_license():
                    err_msg="unable to retrieve license information")
 
         expected_license = {
-            'expires':
-            (datetime.date(2122, 6, 6) - datetime.date.today()).days,
+            'expires': 4813234327,
```

Comment:
There is an inconsistency between `rpk license info` vs `rpk license info --format json`.

In plain text `Expires` is the license's expiration date, but in JSON is the seconds from Unix epoch until license expiration. Can we unify the output for JSON?

Reply:
Nice catch!